### ISSUE TITLE
New set of runs for assessing the climate sensitivity in the Lscale branch

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#496]
### Purpose of the run

We want to start a set of runs to assess the climate sensitivity (4xco2 and 1%co2).
- The baseline for these runs will be 78 (Lscale code branch with faster dycore). 
- We want to use a spunup state for these runs. Unfortunately, 78 is only 49 years and it seems that it might be freezing over towards the end of the run. (@adamrher: can you do your spaghetti plot). 
- We are proposing to start these runs from the end of 75 (Lscale code branch with old dycore) as 75 doesn't seem to freeze over in Adam's plot. 
![Screen Shot 2024-02-15 at 4 32 12 PM](https://github.com/NCAR/amwg_dev/assets/9723220/9743ced9-64fd-43b3-9dce-731f071cd970)

- We will also add a 1xco2 run starting from the same state. 

In summary, we will start a total of 3 runs from the end of 75.
- 78b => BLT1850 starting from 75 at year 99
- 78b_4co2 => 4xco2 starting from 75 at year 99
- 78b_1%co2 => 1%co2  starting from 75 at year 99

In order to start these runs, I need Gustavo's configuration to include the restart from mom6.

### Description of the run

Enter Description

### Suffix in the casename

Suffix

### Namelist modifications

Namelist

### Source modifications

SourceMods

### Sandbox

Requested sandbox or tag

### Contact info

@cecilehannay @PeterHjortLauritzen, @adamrher, @JulioTBacmeister, @brianpm, @swrneale, @islasimpson @dlawrenncar @gustavo-marques

### Any other relevant information

Enter relevant info

**Comment by cecilehannay** [comment]
@dlawrenncar: when I talked to you earlier, we decided to start from the end of 78 but I don't like the way the end of 78 looks.

**Comment by cecilehannay** [comment]
An alternative to this if we really want to start the 78_4xco2 from the 78 restart files (instead of the 75 restart files) is to do a pertlim on 78 and to rerun it hoping that the lab sea won't freeze. And then to start the the 4xco2 from the end of this pertlim run. But this will delay further the start of the 4xco2 runs. 

--- 

I will use the instructions from @gustavo-marques at: https://github.com/NCAR/amwg_dev/discussions/459#discussioncomment-8115445
to make sure the restart files from mom are read properly.

**Comment by adamrher** [comment]
@PeterHjortLauritzen did you add a ```if (things_are_going_well) freeze_the_LabSea``` statement to ur "science optimizations"?

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/4ea982b8-387c-40aa-8bed-f81d912eca9b)

**Comment by PeterHjortLauritzen** [comment]
I agree with proceeding with @cecilehannay's original proposal (and fingers crossed that 78b won't freeze over):

- 78b => BLT1850 starting from 75 at year 99
- 78b_4co2 => 4xco2 starting from 75 at year 99
- 78b_1%co2 => 1%co2 starting from 75 at year 99
